Library/ViewModels/CommentCellViewModelTests.swift

```diff
@@ -299,6 +302,21 @@ internal final class CommentCellViewModelTests: TestCase {
     self.authorBadge.assertValues([.creator, .creator], "The author's badge is emitted.")
   }
 
+  func testBindStylesEmitsCommentStatus() {
+    self.commentStatus.assertDidNotEmitValue()
+
+    let comment = Comment.template
+      |> \.status .~ .retrying
+
+    self.vm.inputs.configureWith(comment: comment, project: .template)
+
+    self.commentStatus.assertValues([.retrying], "The comment status is emitted.")
+
+    self.vm.inputs.bindStyles()
+
+    self.commentStatus.assertValues([.retrying, .retrying], "The comment status is emitted.")
```

Comment:
Can you test a change of status like going from `.retrying` to `.retrySuccess`? Just makes the test more robust when we know we emit more than one type of value. Also we can update the test name to reflect this.

Reply:
Good point, I added this and removed it again because I though we are only testing the output, and the logic that changes the state is in the `CommentsViewModel`, do you think there's still a need?

Reply:
No that's fine. 👍🏿